ayushi/ayushi_postgresql_assignment/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "ayushi_postgresql_assignment",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.ts",
```

Comment:
i don't see index.ts at your folder root. This should point to your entry point file.